logos/brand-guidelines.md

```diff
@@ -0,0 +1,80 @@
+# Resources and Guidelines for TiDB Community
+
+These resources and guidelines were designed to assist the TiDB Community to effectively and consistently build and protect the community’s brand 
+through visual communication.
```

Comment:
Change to:
The following resources and guidelines aim to assist ... the community's brand ...

And pay attention to the punctuation.